packages/presentation/src/file.ts

```diff
@@ -284,3 +284,18 @@ async function uploadFileWithSignedUrl (file: File, uuid: string, uploadUrl: str
     })
   }
 }
+
+export async function fetchJson (file: string, name: string): Promise<Blob | undefined> {
+  const resp = await fetch(getFileUrl(file, name), { signal: AbortSignal.timeout(5 * 1000) })
```

Comment:
I like this, but for a generic utility function the signal should be passed from outside to make it possible to change timeout or use custom signal.

Reply:
Thanks, fixed.